bpython/test/test_autocomplete.py

```diff
@@ -245,12 +245,12 @@ def test_att_matches_found_on_old_style_instance(self):
                                              locals_={'a': OldStyleFoo()}),
                             set(['a.method', 'a.a', 'a.b']))
         self.assertIn(u'a.__dict__',
-                      self.com.matches(3, 'a._', locals_={'a': OldStyleFoo()}))
+                      self.com.matches(3, 'a.__', locals_={'a': OldStyleFoo()}))
```

Comment:
The tests pass with the value set to 3. I also think that makes sense with it finding `a`, `__init__` and `method`. Which attribute would you say is missing?